crates/agglayer-storage/src/storage/migration/test/corrupt.rs

```diff
@@ -94,3 +97,42 @@ fn unexpected_schema() -> Result<(), eyre::Error> {
 
     Ok(())
 }
+
+#[test_log::test]
+fn write_to_readonly_cf_during_migration() -> Result<(), eyre::Error> {
+    let temp_dir = TempDBDir::new();
+    let db_path = &temp_dir.path;
+
+    // Phase 1: Initialize database with V0 schema and add some data
+    {
+        let db = Builder::open_sample(db_path)?.finalize(CFS_V0)?;
+        db.put::<NetworkInfoV0Column>(&NetworkId::new(42), &DATA_V0[0].1)?;
+    }
+
+    // Phase 2: Try to migrate but write to the old (read-only) CF
+    {
+        let result = Builder::open_sample(db_path)?
+            .add_cfs([NetworkInfoV1Column::COLUMN_FAMILY_NAME], |db| {
+                // This should FAIL - trying to write to old CF during migration
+                let v0_value = &DATA_V0[1].1;
+                db.put::<NetworkInfoV0Column>(&NetworkId::new(42), v0_value)?;
+                Ok(())
```

Comment:
Nit: just return `db.put()` result

Reply:
> Nit: just return `db.put()` result

The error type is slightly different.

Edit: Actually, let me double check 😄

Reply:
Yea, I thought this referred to a different bit of code, changed now. Thanks.